app/javascript/app/components/country/country-climate-vulnerability/country-climate-vulnerability-component.jsx

```diff
@@ -108,7 +108,11 @@ class CountryClimateVulnerability extends PureComponent {
                       {section.sectionType === 'LINK' ? (
                         <PrepCard className={styles.prepCard} />
                       ) : (
-                        <Card title={section.title} subtitle={section.subtitle}>
+                        <Card
+                          title={section.title}
+                          subtitle={section.subtitle}
+                          contentFirst
```

Comment:
I prefer to have a variant="contentFirst" or "reverse". What do you think?

Reply:
In this case, I'd rather stay with `contentFirst` flag, because we don't have a third option here, it's also how it's done in `cw-components`. The best option would be to extract the header component and build each card by passing children to the Card component in the right order, so we can build each card from pieces that we want.

Reply:
Mmm is, ok. Leave it this way 👍